doc/whats_new/v0.20.rst

```diff
@@ -376,6 +376,7 @@ Imputer
 - Deprecate :class:`preprocessing.Imputer` and move the corresponding module to
   :class:`impute.SimpleImputer`. :issue:`9726` by :user:`Kumar Ashutosh
   <thechargedneutron>`.
+- The ``axis`` parameter was deprecated in this version.
```

Comment:
deprecated in -> dropped from (or removed from)

Please remove the `-` at the beginning, and in the next line also. We want readers to see all these points as one.

Reply:
You can also credit yourself for this work, and reference thus pull request